subiquitycore/ui/buttons.py

```diff
@@ -27,12 +27,30 @@ class MenuSelectButton(Button):
     button_right = Text(">")
 
 
-start_btn = partial(PlainButton, label="Start", on_press=None)
-confirm_btn = partial(PlainButton, label="Confirm", on_press=None)
-cancel_btn = partial(PlainButton, label="Cancel", on_press=None)
-done_btn = partial(PlainButton, label="Done", on_press=None)
-finish_btn = partial(PlainButton, label="Finish", on_press=None)
-ok_btn = partial(PlainButton, label="OK", on_press=None)
-continue_btn = partial(PlainButton, label="Continue", on_press=None)
-reset_btn = partial(PlainButton, label="Reset", on_press=None)
-menu_btn = partial(MenuSelectButton, on_press=None)
+def plain_btn(label, color, on_press=None, user_arg=None):
+    button = PlainButton(label=label)
+    if on_press is not None:
+        connect_signal(button, 'click', on_press, user_arg)
+    return AttrWrap(button, color, color + ' focus')
+
+
+start_btn = partial(plain_btn, label="Start", color="save_button")
```

Comment:
Ha, ok. this is simple enough. I thought we were going to do something crazy like:
```
class StartBtn(AttrWrap)
    def __init__(on_press=None, user_arg=None):
         w = PlainButton(label='Start', on_press=on_press)
         super(self).__init__(w=w, "save_button", "save_button focus")
```
Or some such, but your functional implementation is fine as well.

Using classes would mean that we would be able to drop `partial(...)` completely. As this partial stuff, is like fake class / obscure lambda function =)
I guess moving to classes can be done as another refactor. This at least makes all buttons semantically color coded straight away.

Reply:
Yeah, I'm not particularly happy about this but it works.

I think we need to be able to override the labels.